server/fleet/mdm.go

```diff
@@ -33,6 +33,17 @@ func (a AppleBM) AuthzType() string {
 	return "mdm_apple"
 }
 
+type AppleCSR struct {
+	// NOTE: []byte automatically JSON-encodes as a base64-encoded string
```

Comment:
heh thanks for the note, the first time I faced this I was _deeply_ confused

Reply:
Same! It'd be a nice Go trick-question :D